services.py

```
@@ -0,0 +1,116 @@
+        plot_data = {"x": x, "y": y}
+        self.element_internal_op(widget_id, json.dumps(plot_data))
+
+    def set_plot_line_axes_decimal_digits(self, widget_id: int, x: float, y: float):
+        axes_data = {"x": x, "y": y}
+        self.element_internal_op(widget_id, json.dumps(axes_data))
+
+    def append_text_to_clipped_multi_line_text_renderer(self, widget_id: int, text: str):
+        self.extern_append_text(widget_id, text)
+
+    def set_input_text_value(self, widget_id: int, value: str):
+        input_text_data = {"value": value}
```

Ruff (0.8.2)

Local variable `input_text_data` is assigned to but never used

Remove assignment to unused variable `input_text_data`

(F841)